low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,23 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
```

Comment:
There is no kafka broker in the list of monitored services?
@ujjwalpl  Could you please review the list. Some of services look unnecessary.

Reply:
>> There is no kafka broker in the list of monitored services?
Currently sytemd interface is not available for kafka service, created tkt for foundation team([EOS-18476](https://jts.seagate.com/browse/EOS-18476)) to provide systemd interfafce for this service. Once it is available we will add in monitored_list.

Reply:
Any other 3rd party service beyond 'Kibana' which is managed by PCS @ajaysrivas ?

> Some of services look unnecessary.

This list will evolve for missing services. And also getting reviewed to remove any discrepancies only